src/components/e-button.vue

```diff
@@ -127,6 +127,14 @@
         type: Boolean,
         default: false,
       },
+
+      /**
+       * Overwrites the element of the button component.
```

Comment:
Please add a hint, that this option should only be used, when using the button component within e.g. a html anchor tag to make sure to prevent standalone buttons being `span` or similiar